### PR TITLE
Split hardware requires in the metainfo file

### PR DESCRIPTION
**Pull request by hughsie**
Fixes: https://github.com/hughsie/lvfs-website/issues/225

### CODE REVIEW

File: app/views_upload.py [hunk 1]
```diff
@@ -111,12 +111,18 @@ def _create_fw_from_uploaded_file(ufile):
...
-                             AppStreamGlib.Require.kind_to_string(req.get_kind()),
-                             req.get_value(),
-                             AppStreamGlib.Require.compare_to_string(req.get_compare()),
-                             req.get_version())
-            md.requirements.append(rq)
+            req_kind = AppStreamGlib.Require.kind_to_string(req.get_kind())
+            req_compare = AppStreamGlib.Require.compare_to_string(req.get_compare())
+            req_value = req.get_value()
+            if req_kind == 'hardware':
+                req_values = req_value.split('|')
```
Comment (superm1) [inline app/views_upload.py]: I think you should also add this to documentation so it doesn't become a magic delimiter that you only find out about by checking source.

Comment (hughsie) [reply app/views_upload.py]: Documentation in fwupd, or lvfs-website (e.g. metainfo pages)?

Comment (superm1) [reply app/views_upload.py]: I think LVFS website, that's where the example for this is right?

Comment (hughsie) [reply app/views_upload.py]: Already there I think: https://fwupd.org/lvfs/docs/metainfo/restrict

Comment (superm1) [reply app/views_upload.py]: Great!

**Pull request merged**